### PR TITLE
Mussel 0.1.5

### PR DESCRIPTION
**Pull request by renatogamboa**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by godrei** [comment]
Hi @renatogamboa ,
thanks for the contribution!

I'm trying to test this step, could you please explain how should I use this on Bitrise/locally? Which steps are needed to run before this one?

**Comment by renatogamboa** [comment]
Hi @godrei,
Thanks for reviewing and testing!

The best way to use this step is by adding the [Mussel](https://github.com/UrbanCompass/Mussel) framework to your XCode Project/App via cocoapods. After pod installing, running this step will launch the `Mussel Server App` on bitrise's virtual machine. This allows developers to make local requests to the Mussel Server which triggers either Push Notifications or Universal Links on the iOS Simulator during UITests.

File: steps/mussel/0.1.5/step.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+project_type_tags:
+- ios
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/mussel/0.1.5/step.yml:43]: This is not used by script

**Comment by koral--** [comment]
So that pod should be probably (tried to) be installed by the script.
Also your commits should be squashed into single one.

**Pull request closed without merging**